### ISSUE TITLE
Mac Pupil Capture v2.0 won't launch

### ISSUE THREAD

**Issue by m-macaskill** [pupil-labs/pupil#1919]
The Pupil Capture app won't open on my Mac. Pupil Player and Pupil Service both launch OK. I can see that Pupil Capture launches and that it creates three processes, but then Activity Monitor shows that one of them has become non-responsive, and no interface ever appears. 
Latest version (2.0) of all three Pupil apps. MacBook Pro, macOS Mojave 10.14.6, 16 GB RAM. 

When trying to run run the Pupil Capture app from the command line, this output was shown in the Terminal:

```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "PyInstaller/loader/pyiboot01_bootstrap.py", line 151, in __init__
  File "ctypes/__init__.py", line 348, in __init__
OSError: dlopen(libSystem.dylib, 6): image not found

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "threading.py", line 916, in _bootstrap_inner
  File "threading.py", line 864, in run
  File "pyre/zactor.py", line 59, in run
  File "pyre/zbeacon.py", line 66, in __init__
  File "pyre/zbeacon.py", line 303, in run
  File "pyre/zbeacon.py", line 218, in handle_pipe
  File "pyre/zbeacon.py", line 201, in configure
  File "pyre/zbeacon.py", line 73, in prepare_udp
  File "pyre/zbeacon.py", line 137, in _prepare_socket
  File "pyre/zhelper.py", line 232, in get_ifaddrs
  File "PyInstaller/loader/pyiboot01_bootstrap.py", line 153, in __init__
PyInstallerImportError: Failed to load dynlib/dll 'libSystem.dylib'. Most probably this dynlib/dll was not found when the application was frozen.
```

**Comment by papr** [comment]
Hi @m-macaskill 
Thank you very much for the detailed information. `libSystem.dylib` is [explicitly removed](https://github.com/pupil-labs/pupil/blob/master/deployment/deploy_capture/bundle.spec#L59-L63) during packaging as we have encountered compatibility issues across different macOS versions in the past if it was included the bundle.

Your Mac's `libSystem.dylib` file should be located at `/usr/lib/libSystem.dylib`. Could you please verify this is the case for you?

I personally am not able to reproduce your issue on my Mac. :-/

**Comment by m-macaskill** [comment]
Thanks for looking into this.

On my Mac, `/usr/lib/libSystem.dylib` exists. It is an alias that points to an original at: `/usr/lib/libSystem.B.dylib`

**Comment by papr** [comment]
@m-macaskill We have updated the bundles. Could you please give them a try?

**Comment by m-macaskill** [comment]
Thanks. I've downloaded the .dmg and installed what I think is the latest version of Pupil Capture (2.0.182). But unfortunately the problem still occurs, with the same error message.

I've since downloaded an old version: 1.23.5 does run OK.

**Comment by papr** [comment]
@m-macaskill We were only able to reproduce it the issue on macOS Mojave 10.14.6 (your version). We have also tested macOS Catalina and macOS High Sierra. On both OS, the app works as expected.

**Comment by m-macaskill** [comment]
Thanks very much for investigating this thoroughly and isolating it to a particular OS version. That makes it possible to work around.

**Comment by papr** [comment]
The fix has been released as part of the [v2.1 release](https://github.com/pupil-labs/pupil/releases/v2.1).

**Comment by m-macaskill** [comment]
Thank you, can confirm that this fixes the issue for me.
